src/Backend/Modules/Profiles/Engine/Model.php

```diff
@@ -37,10 +37,10 @@ class Model
      * @var string
      */
     const QUERY_DATAGRID_BROWSE_PROFILE_GROUPS =
-        'SELECT gr.id, g.name AS group_name, UNIX_TIMESTAMP(gr.expires_on) AS expires_on
+        'SELECT gr.id, g.name AS group_name, UNIX_TIMESTAMP(gr.expires_on) AS expires_on,
+          IF(gr.expires_on IS NOT NULL AND gr.expires_on <= NOW(), 1, 0) AS hidden
```

Comment:
I would add a comment here that you name it hidden to gray it out in the datagrid

Reply:
I've amended it